projects/Mallard/src/components/slider/draw/lozenge.tsx

```diff
@@ -166,13 +166,8 @@ const LozengeBigHeader = ({
                                 opacity: position.interpolate({
                                     inputRange: safeInterpolation([
                                         0,
-                                        clamp(
-                                            width / 2 -
-                                                metrics.fronts.sliderRadius,
-                                            0,
-                                            width / 2,
-                                        ),
-                                        width / 2,
+                                        fadeLozengeAt,
+                                        fadeLozengeAt * 2,
```

Comment:
I just did this so if there was any sort of misalignment then it would fade out rather than snap out quickly ...